main.py

```diff
@@ -1,10 +1,44 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_data in data.items():
+
+        race = player_data["race"]
+        if not Race.objects.filter(name=race["name"]).exists():
+            Race.objects.create(
+                name=race["name"],
+                description=race["description"]
+            )
+
+        for skill in race["skills"]:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=Race.objects.get(name=race["name"])
+                )
+
+        guild = player_data["guild"]
+        if guild and not Guild.objects.filter(name=guild["name"]).exists():
```

Comment:
I don't like the way you check the same `guild` value twice, even though I understand that the code looks better this way.

By the way, it would be more efficient in terms of database interaction to store the return value of `.create()` and use `.get()` only if you need to. But yet again, the code won't be as pretty as it is now.

The other option is to use `.get_or_create()`.

Reply:
Much appreciated!
Thank you, especially for the `.get_or_create()` hint. With it the code will be much shorter and readable.
Guess I'll refactor in between today's kick-offs: